pyproject.toml

```diff
@@ -25,8 +25,19 @@ where = [".hooks"]
 check-conditional-mark-sort = 'pre_commit_hooks.check_conditional_mark_sort:main'
 
 [tool.black]
-exclude = '''
-/(
-    spytest/
-)/
+line-length = 88
```

Comment:
Most people are using wide screen displays nowadays. Can we increase the line-length to 120?